ksqldb-engine/src/main/java/io/confluent/ksql/engine/EngineExecutor.java

```diff
@@ -601,7 +601,7 @@ private ExecutorPlans planQuery(
         query,
         sink,
         metaStore,
-        ksqlConfig
+        ksqlConfig.cloneWithPropertyOverwrite(statement.getSessionConfig().getOverrides())
```

Comment:
This instance of passing `ksqlConfig` around wasn't added as part of the feature flag PR that introduced this bug. Updating this line looks like it has implications beyond the scope of the feature flag?